2017/Day 2-1.py

```diff
@@ -0,0 +1,7 @@
+checksum = 0
+with open("Day 2 - input", "r") as file:
+    for line in file:
+        row = [int(i) for i in line.split()]
+        result = max(row) - min(row)
+        checksum += result
+print(F"The checksum is {checksum}")
```

Comment:
The `f` in `f-strings` should be lowercase by convention

Reply:
yeah, I was about to review this lol..